presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -45,6 +45,7 @@
     private int re2JDfaStatesLimit = Integer.MAX_VALUE;
     private int re2JDfaRetries = 5;
     private RegexLibrary regexLibrary = JONI;
+    private boolean parseDecimalLiteralsAsDouble;
```

Comment:
Until we make decimals faster, I don't think we should turn the new behavior on by default.